L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -3,22 +3,24 @@
 using BinaryBuilder, Pkg
 
 name = "libcxxwrap_julia"
-version = v"0.6.6"
+version = v"0.7.0"
+
+const is_yggdrasil = haskey(ENV, "BUILD_BUILDNUMBER")
+git_repo = is_yggdrasil ? "https://github.com/JuliaInterop/libcxxwrap-julia.git" : joinpath(ENV["HOME"], "src/julia/libcxxwrap-julia/")
+unpack_target = is_yggdrasil ? "" : "libcxxwrap-julia"
 
 # Collection of sources required to complete build
 sources = [
-    "https://github.com/JuliaInterop/libcxxwrap-julia.git" =>
-    "c1c54414ed8875f963253c8b2e4c960f92e32908",
+    GitSource(git_repo, "7959f09947f95e67ee094a2ee3fb8b7f9be696f8", unpack_target=unpack_target),
+    ArchiveSource("https://julialang-s3.julialang.org/bin/linux/armv7l/1.3/julia-1.3.1-linux-armv7l.tar.gz", "lala"; unpack_target="julia"),
```

Comment:
You have to use different unpack targets :wink: For example use the triplet of the target platform, or `julia-${triplet}`

Reply:
Yes, and my hope that putting "lala" as hash to trick it into spitting out the correct hash codes in one go was also squashed :)